### PR TITLE
Adds dollar signs to expense reports

### LINKED ISSUE #593: Add dollar signs to anticipated expenses on reviewer screen / show page

### PR DESCRIPTION
**Pull request by sdellis**
Fixes #593 

Requires LUX 2.16.1

### CODE REVIEW

File: app/views/travel_requests/_review_form.html.erb [hunk 1]
```diff
@@ -74,6 +74,7 @@
           <card-content>
             <data-table
               caption="Anticipated Expenses"
+              class="estimates"
```
Comment (carolyncole) [inline app/views/travel_requests/_review_form.html.erb]: This line should be combined with the already existing class on line 79

File: app/views/travel_requests/show.html.erb [hunk 2]
```diff
@@ -60,6 +60,7 @@
       <card size="full-width">
         <card-content>
           <data-table
+            class="estimates"
```
Comment (carolyncole) [inline app/views/travel_requests/show.html.erb]: @sdellis this one also should be combined with line 67

File: app/views/travel_requests/_review_form.html.erb (hunk 1 above)
Comment (sdellis) [reply app/views/travel_requests/_review_form.html.erb]: Removed the overriding class

File: app/views/travel_requests/show.html.erb (hunk 2 above)
Comment (sdellis) [reply app/views/travel_requests/show.html.erb]: Removed the overriding class

**Pull request merged**